src/lib/operators/print.cpp

```diff
@@ -73,10 +73,6 @@ std::shared_ptr<const Table> Print::_on_execute() {
   // print each chunk
   for (ChunkID chunk_id{0}; chunk_id < input_table_left()->chunk_count(); ++chunk_id) {
     auto chunk = input_table_left()->get_chunk(chunk_id);
-    if (chunk->size() == 0 && (_flags & PrintIgnoreEmptyChunks)) {
```

Comment:
So the flag is removed which disables the printing of "empty chunk.".

Do I get this PR right? : we cannot have empty chunks except at the beginning of a table's lifetime, hence we always print the info as it helps us to recognize empty tables. As soon as one row has been added, the "empty chunk" info will never be display again anyhow.

Correct?

Reply:
No. The option of *ignoring* empty chunks is removed.

Reply:
Yeah, that's what I tried to say.